test/integration/features/addons/addon-htpasswd-identity-provider.feature

```diff
@@ -0,0 +1,105 @@
+@addon-htpasswd-identity-provider @core
+Feature: Configures single identity for OpenShift login instance
+This add-on configures OpenShift to use only a single identity provider
+
+  Scenario: User can enable the htpasswd-identity-provider
+     When executing "minishift addons enable htpasswd-identity-provider" succeeds
+     Then stdout should contain "Add-on 'htpasswd-identity-provider' enabled"
+
+  Scenario: User can start Minishift with htpasswd-identity-provider add-on enabled
+    Given Minishift has state "Does Not Exist"
+      And image caching is disabled
+     When executing "minishift start" succeeds
+     Then Minishift should have state "Running"
+      And stdout should contain
+      """
+      Successfully installed addon htpasswd identity provider
+      """
+
+  Scenario: User can login to openshift instance only with default username "developer" and password "developer"
+    Given Minishift has state "Running"
+      And user waits "60" seconds
+     When executing "oc login -u developer -p developer" succeeds
```

Comment:
I will recommend using retry mechanism instead of `wait`.

Reply:
Right, retry logic is like life and time saver.